executor/agents/agno/config_utils.py

```
@@ -59,16 +60,35 @@ def resolve_value_from_source(
+
+                if (
+                    dict_from_object is not None
+                    and isinstance(dict_from_object, dict)
+                    and key in dict_from_object
+                ):
+                    current = dict_from_object[key]
+                else:
+                    return ""
 
         return str(current) if current is not None else ""
     except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)